src/components/slider/slider.tsx

```diff
@@ -15,31 +15,58 @@ import {
     styleUrl: 'slider.scss',
 })
 export class Slider {
+    /**
+     * Set to `true` id input is disabled
```

Comment:
Is there a typo in this description, or am I just having a brain freeze? 😄

Reply:
Ah! "id" should be "if", right? 😂

```suggestion
     * Set to `true` to disable the input
```

maybe?

Reply:
Haha yeah there was a typo, but your version is good as well :smile: